src/app/accounts/send-default-wallet-balance-to-users.ts

```diff
@@ -26,7 +26,7 @@ export const sendDefaultWalletBalanceToAccounts = async () => {
       const balanceAmount = await LedgerService().getWalletBalanceAmount(wallet)
       if (balanceAmount instanceof Error) return balanceAmount
 
-      const displayPriceRatio = await getCurrentPriceAsPriceRatio({
+      const displayPriceRatio = await getCurrentPriceAsWalletPriceRatio({
```

Comment:
same here `getCurrentPriceAsDisplayPriceRatio` then we will change this to account display currency (which is display currency)

Reply:
https://github.com/GaloyMoney/galoy/pull/2334#discussion_r1108847733